### PR TITLE
Some fixes to emoji/ANSI renderer

### PR DESCRIPTION
**Pull request by mikehearn**
Catches up with a logging change in master.

### CODE REVIEW

**Comment by mikehearn** [comment]
Going to do a bit more on this ...

File: node/src/main/kotlin/net/corda/node/utilities/ANSIProgressRenderer.kt [hunk 1]
```diff
@@ -127,8 +127,7 @@ object ANSIProgressRenderer {
         var newLinesDrawn = 1 + pt.renderLevel(ansi, 0, error != null)
 
         if (error != null) {
-            // TODO: This should be using emoji only on supported platforms.
-            ansi.a("$CODE_SKULL_AND_CROSSBONES   $error")
+            ansi.a("${Emoji.skullAndCrossbones}   ${error.message}")
```
Comment (shamsasari) [inline node/src/main/kotlin/net/corda/node/utilities/ANSIProgressRenderer.kt]: `Throwable.toString()` includes the exception class which would be useful here, no?

Comment (mikehearn) [inline node/src/main/kotlin/net/corda/node/utilities/ANSIProgressRenderer.kt]: Progress trackers are meant to be viewed by human operators, not developers (or not so much).

**Comment by mikehearn** [comment]
@shamsasari OK, it's ready for your review now - sorry, I took you off it but you reviewed it anyway. Can you PTAL?

File: core/src/main/kotlin/net/corda/core/utilities/Emoji.kt [hunk 2]
```diff
@@ -7,7 +7,9 @@ import net.corda.core.codePointsString
  */
 object Emoji {
     // Unfortunately only Apple has a terminal that can do colour emoji AND an emoji font installed by default.
-    val hasEmojiTerminal by lazy { listOf("Apple_Terminal", "iTerm.app").contains(System.getenv("TERM_PROGRAM")) }
+    val hasEmojiTerminal by lazy {
+        System.getenv("CORDA_FORCE_EMOJI") != null || System.getenv("TERM_PROGRAM") in listOf("Apple_Terminal", "iTerm.app")
```
Comment (shamsasari) [inline core/src/main/kotlin/net/corda/core/utilities/Emoji.kt]: What's CORDA_FORCE_EMOJI?

Comment (mikehearn) [inline core/src/main/kotlin/net/corda/core/utilities/Emoji.kt]: Just a way to let us test the code on other platforms/terms without having to recompile. Nothing special.

**Pull request merged**